unittests/Sema/SemaFixture.h

```diff
@@ -0,0 +1,68 @@
+//===--- SemaFixture.h - Helper for setting up Sema context -----*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/AST/ASTContext.h"
+#include "swift/AST/DiagnosticEngine.h"
+#include "swift/AST/Module.h"
+#include "swift/AST/SourceFile.h"
+#include "swift/AST/Type.h"
+#include "swift/Basic/LangOptions.h"
+#include "swift/Basic/Platform.h"
+#include "swift/Basic/SourceManager.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/ADT/SmallString.h"
+#include "llvm/Support/Host.h"
+#include "llvm/Support/Path.h"
+#include "gtest/gtest.h"
+#include <string>
+
+namespace swift {
+namespace unittest {
+
+class SemaTestBase : public ::testing::Test {
+public:
+  LangOptions LangOpts;
+  TypeCheckerOptions TypeCheckerOpts;
+  SearchPathOptions SearchPathOpts;
+  ClangImporterOptions ClangImporterOpts;
+  SourceManager SourceMgr;
+  DiagnosticEngine Diags;
+
+  SemaTestBase() : Diags(SourceMgr) {
+    LangOpts.Target = llvm::Triple(llvm::sys::getProcessTriple());
+
+    llvm::SmallString<128> libDir(SWIFTLIB_DIR);
+    llvm::sys::path::append(libDir, getPlatformNameForTriple(LangOpts.Target));
```

Comment:
I'm guessing that this doesn't work on Windows same way it does on macOS where stdlib module is located in `lib/swift/macosx`?

Reply:
There is a piece of code in `lit.cfg` that computes the stdlib path (including on Windows) that seem to work:
https://github.com/apple/swift/blob/main/test/lit.cfg#L1495

Reply:
The layout on Windows is similar to that on macOS: `lib/swift/windows/Swift.swiftmodule/x86_64-unknown-windows-msvc.swiftmodule`.

Reply:
Thanks to @compnerd, we have figured that I should have used default target instead. Now everything is fine.